sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestRunClient.java

```diff
@@ -334,16 +253,15 @@ public Response<BinaryData> createOrUpdateAppComponentsWithResponse(
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @return the response to a metrics query as paginated response with {@link PagedIterable}.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public PagedIterable<BinaryData> listMetrics(
             String testRunId,
-            String metricname,
+            String metricName,
             String metricNamespace,
             String timespan,
             RequestOptions requestOptions) {
         return new PagedIterable<>(
-                this.client.listMetrics(testRunId, metricname, metricNamespace, timespan, requestOptions));
+                this.client.listMetrics(testRunId, metricName, metricNamespace, timespan, requestOptions));
```

Comment:
nit, this might actually benefit from a [x-ms-client-name](https://azure.github.io/autorest/extensions/#x-ms-client-name), either in readme.md if only for Java, or in swagger if for all SDK (instead of customization here).

Reply:
Updated

Reply:
Done